docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,629 @@
+    "- **physical type**: `int64`\n",
+    "- **standard tags**: `{'numeric'}`\n",
+    "\n",
+    "#### AgeNullable\n",
+    "\n",
+    "Represents Logical Types that contain non-negative numbers indicating a person’s age. May contain null values.\n",
+    "- **physical type**: `Int64`\n",
+    "- **standard tags**: `{'numeric'}`\n",
+    "\n",
+    "#### Double\n",
+    "\n",
+    "Represents Logical Types that contain positive and negative numbers, some of which include a fractional component.\n",
+    "- **physical type**: `float64`\n",
+    "- **standard tags**: `{'numeric'}`\n",
+    "\n",
+    "#### Integer\n",
+    "\n",
+    "Represents Logical Types that contain positive and negative numbers without a fractional component, including zero (0).\n",
+    "- **physical type**: `int64`\n",
+    "- **standard tags**: `{'numeric'}`\n",
+    "\n",
+    "#### IntegerNullable \n",
+    "Represents Logical Types that contain positive and negative numbers without a fractional component, including zero (0). May contain null values. \n",
+    "- **physical type**: `Int64`\n",
+    "- **standard tags**: `{'numeric'}`\n",
+    "\n",
+    "\n",
+    "\n",
+    "Below we'll find a dataframe with examples of each of the numeric LogicalTypes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "handled-russia",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "numerics_df = pd.DataFrame({\n",
+    "    'ints' : [1, 2, 3, 4],\n",
+    "    'ints_nullable': pd.Series([1, 2, None, 4], dtype='Int64'),\n",
+    "    'floats' : [0.0, 1.1, 2.2, 3.3],\n",
+    "    'ages': [18, 22, 24, 34],\n",
+    "    'ages_nullable' : [None, 2, 22, 33]\n",
+    "})\n",
+    "\n",
+    "numerics_df.ww.init(logical_types={'ages':'Age', 'ages_nullable':'AgeNullable'})\n",
+    "numerics_df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "forty-banks",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Categorical Logical Types\n",
+    "\n",
+    "#### Categorical \n",
+    "\n",
+    "Represents a Logical Type with few unique values relative to the size of the data.\n",
+    "\n",
+    "- **physical type**: `category`\n",
+    "- **inference**: Woodwork defines a threshold for percentage unique values relative to the size of the series below which a series will be considered categorical. See [setting config options guide](https://woodwork.alteryx.com/en/stable/guides/setting_config_options.html#Categorical-Threshold) for more information on how to control this threshold.\n",
+    "- **koalas note**: Koalas does not support the `category` dtype, so for Koalas DataFrames and Series, the `string` dtype will be used.\n",
+    "\n",
+    "\n",
+    "Some examples of data for which the Categorical logical type would apply:\n",
+    "\n",
+    "- Gender\n",
+    "- Eye Color\n",
+    "- Nationality\n",
+    "- Hair Color\n",
+    "- Spoken Language\n",
+    "\n",
+    "#### CountryCode\n",
```

Comment:
With CountryCode, we want to tell users to use the ISO format. See here
https://github.com/alteryx/featuretools/blob/main/featuretools/variable_types/variable.py#L410
We should update the CountryCode docstring as well.

Reply:
So we want the length-2 codes (US) and not the length-3 ones (USA)? The example in featuretools specifies 2 and has examples that match. the Woodwork docstring examples for countrycode have always used 3. (though the sub region code uses 2)

Reply:
We want to use the length-2 codes (ISO 3166-1 alpha-2), so lets update the Woodwork docstring to have that.

Reply:
added the ISO 2166 requirements to the docstrings and sections of the guide